.github/workflows/validate-new-issue.yml

```diff
@@ -9,7 +9,22 @@ jobs:
     - uses: actions/checkout@v2
     - name: "Validate issue against templates"
       shell: bash
+      env:
+        GITHUB_TOKEN: ${{ github.token }}
       run: |
+        # Trust users who belong to the getsentry org (id: 1396951). Due to API
+        # limitations, we can only see this if the user's membership is public.
+        sentry_org=$(
+          gh api "https://api.github.com/users/${{ github.actor }}/orgs" \
+            | jq '.[] | select(.id==1396951)'
+        )
+        if [ -z "$sentry_org" ]; then
+          echo "${{ github.actor }} is not a member of the getsentry org. 🧐"
```

Comment:
```suggestion
          echo "::INFO::${{ github.actor }} is not a member of the getsentry org. 🧐"
```

Reply:
I tried this out but it isn't having the expected result.

<img width="857" alt="Screen Shot 2021-01-15 at 9 07 40 AM" src="https://user-images.githubusercontent.com/134455/104741189-ddfa5680-5716-11eb-8839-d54867d5633c.png">

Turns out `::info` is [not a supported prefix](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-a-warning-message).